src/main/java/org/fundacionjala/at15/katas/langtonant/agustin/Ant.java

```diff
@@ -0,0 +1,115 @@
+package org.fundacionjala.at15.katas.langtonant.agustin;
+
+public class Ant {
+    // TODO: Make Ant Class with constructor and methods
+
+    // variables
+    int xPos;
+    int yPos;
+    // direction
+    String direction;
+    Grid g;
+    boolean isFinished;
+
+    public Ant(int xPos, int yPos, String direction, Grid g) {
```

Comment:
Try to avoid variable names with one single letter, they are not descriptive enough to understand what kind of value they store

Reply:
Ok! refactoring right now. I will take it into consideration in the future.